README.md

```diff
@@ -5,7 +5,19 @@ All source code of models (bundles) are tracked in `models/`, and for each disti
 
 ## Getting Started
 
-To download and get started with the models, please see [the model browser](https://monai.io/model-zoo.html) and [the example use cases](https://github.com/Project-MONAI/tutorials/tree/main/model_zoo).
+A basic example to download and extract a bundle is:
+
+```bash
+pip install "monai[fire]"
+python -m monai.bundle download wholeBody_ct_segmentation --bundle_dir "bundles/"
+```
+
+- The commands will download `wholeBody_ct_segmentation` to the current directory's `bundles/` subdirectory.
+- For more downloading options, please run `python -m monai.bundle download -h`
+- For the specific usage of a bundle, please refer to its `docs` folder, for example, `bundles/wholeBody_ct_segmentation/docs`.
+
+To browse the available models, please see [the model browser](https://monai.io/model-zoo.html).
```

Comment:
I would suggest to move this line to the beginning of this section, users usually browse the bundles then download.
And BTW, suggest: "available models" -> "available bundles".

Thanks.

Reply:
from that page, I couldn't find any basic commands for how to set things up.

Reply:
And BTW, suggest: "available models" -> "available bundles".
> 

given this repo is a 'model' zoo, I think available 'models' make sense.

Reply:
> from that page, I couldn't find any basic commands for how to set things up.

Yeah, @zephyrie @yiheng-wang-nv I think we may consider add the description of how to download in https://monai.io/model-zoo.html?

Thanks in advance.

Reply:
Maybe we can refer to Hugging Face's `How to clone` button, and enhance our `download` button:
![Screen Shot 2023-02-27 at 2 17 11 PM](https://user-images.githubusercontent.com/68361391/221489121-028d1dae-7dd7-4f1d-8d6f-a42655699173.png)

The following is from their [webpage](https://huggingface.co/MONAI/example_spleen_segmentation)
![Screen Shot 2023-02-27 at 2 20 05 PM](https://user-images.githubusercontent.com/68361391/221489226-96160d19-ccb7-4191-9dee-1440b9ce923c.png)
and we can insert the code @wyli added automatically. The only variable is the bundle name.

Reply:
Hi @Nic-Ma , I just create a ticket in:
https://github.com/Project-MONAI/project-monai.github.io/issues/46
As for the potential change in model zoo webpage, maybe we can discuss there, and this PR can be merged first as it does enhances the readme. What do you think?

Reply:
Hi @wyli ,

As @yiheng-wang-nv already plan to enhance the web page, can we move the "browse the available models" line to the beginning?

Thanks.